pinot-plugins/pinot-input-format/pinot-csv/src/main/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordExtractor.java

```diff
@@ -69,4 +70,32 @@ public GenericRow extract(CSVRecord from, GenericRow to) {
     }
     return to;
   }
+
+  @Override
+  @Nullable
+  public Object convert(@Nullable Object value) {
```

Comment:
I think you missed removing this code from `extract` and also calling `convert`  above in the `extract`.

Reply:
You're right, good catch.